ServerlessDashboard/Models/TestComparisonModel.cs

```diff
@@ -0,0 +1,85 @@
+        public string PlatformDiff => FirstTest.Platform == SecondTest.Platform ? "Same" : "Different";
+        public string SucceededExecutionsDiff => (FirstTest.SucceededExecutions - SecondTest.SucceededExecutions).ToString();
+        public string FailedExecutionsDiff => (FirstTest.FailedExecutions - SecondTest.FailedExecutions).ToString();
+        public string TimeoutExecutionsDiff => (FirstTest.TimeoutExecutions - SecondTest.TimeoutExecutions).ToString();
+        public string AverageLatencyDiff => (FirstTest.AverageLatency - SecondTest.AverageLatency).ToString();
+        public string TargetEpsDiff => (FirstTest.TargetEps - SecondTest.TargetEps).ToString();
+        public string ExecutionCountDiff => (FirstTest.ExecutionCount - SecondTest.ExecutionCount).ToString();
+        public string ExecutionTimeStdDiff=> (FirstTest.ExecutionTimeStandardDeviation - SecondTest.ExecutionTimeStandardDeviation).ToString();
+        public string FunctionClockTimeDiff => (FirstTest.FunctionClockTime - SecondTest.FunctionClockTime).ToString();
+        public string HostConcurrencyDiff => (FirstTest.HostConcurrency - SecondTest.HostConcurrency).ToString();
+        public string ThroughputDiff => (FirstTest.Throughput - SecondTest.Throughput).ToString();
+        public string ErrorsDiff => (FirstTest.Errors - SecondTest.Errors).ToString();
+
+        public TestComparisonModel(Test firstTest, Test secondTest)
+        {
+            FirstTest = new TestResultsModel(firstTest);
+            SecondTest = new TestResultsModel(secondTest);
+            ComparedThroughput = GetResults((x) => x.SuccessCount, "Throughput");
+            ComparedLatency = GetResults((x) => x.AverageLatency, "Latency");
+            ComparedErrorCount = GetResults((x) => x.FailedCount, "Errors");
+            ComparedTimeoutCount = GetResults((x) => x.TimeoutCount, "Timeouts");
+            ComparedHostConcurrencyCount = GetResults((x) => x.HostConcurrency, "HostConcurrency");
+        }
+
+        private List<object> GetResults(Func<TestResult, object> selector, string prefixName)
+        {
+            var firstResults = this.FirstTest.TestResults;
+            var secondResults = this.SecondTest.TestResults;
+            var timeDiff = (firstResults.First().Timestamp - secondResults.First().Timestamp).TotalSeconds;
+
+            return new List<object>
+            {
+                new
+                {
+                    label = prefixName + this.FirstTest.Name.Substring(8) + this.FirstTest.Id,
+                    data = firstResults.Select(x => new object[] { ToFlotTimestamp(x.Timestamp), selector(x) }).ToList()
+                },
+                new
+                {
+                    label = prefixName + this.FirstTest.Name.Substring(8) + this.SecondTest.Id,
+                    data = secondResults.Select(x => new object[] { ToFlotTimestamp(x.Timestamp.Add(TimeSpan.FromSeconds(timeDiff))), selector(x) }).ToList()
```

Comment:
For timeDiff here, you make a TimeSpan above, then you take just the total seconds and then you make a new timespan using those seconds. I don't understand what this is achieving, because totalSeconds is a double, so there is no truncation. Why not just:

```
var timeDiff = (firstResults.First().Timestamp - secondResults.First().Timestamp)
...
x.Timestamp.Add(timeDiff)
```